client/components/videos-ui/index.jsx

```diff
@@ -118,26 +145,18 @@ const VideosUi = ( {
 				) }
 				<div className="videos-ui__header-content">
 					<div className="videos-ui__titles">
-						<h2>{ translate( 'Watch five videos.' ) }</h2>
-						<h2>{ translate( 'Save yourself hours.' ) }</h2>
+						<h2>{ headerTitle ? headerTitle : translate( 'Watch five videos.' ) }</h2>
+						<h2>{ headerSubtitle ? headerSubtitle : translate( 'Save yourself hours.' ) }</h2>
```

Comment:
I think we can be optimistic here and always assume that `headerTitle` and `headerSubtitle ` would be set. In the worst case scenario we won't be display a header if someone forgets to set fit for a new course, but I guess that's better than showing a totally unrelated title 🙂 
```suggestion
						<h2>{ headerTitle }</h2>
						<h2>{ headerSubtitle }</h2>
```